-------------------------------------------------------------------------------
Test set: com.wzf.RedisTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 25.878 s <<< FAILURE! - in com.wzf.RedisTest
testGet(com.wzf.RedisTest)  Time elapsed: 16.986 s  <<< ERROR!
org.springframework.data.redis.RedisConnectionFailureException: Unable to connect to Redis; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6379
	at com.wzf.RedisTest.testGet(RedisTest.java:31)
Caused by: io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6379
	at com.wzf.RedisTest.testGet(RedisTest.java:31)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
Caused by: java.net.ConnectException: Connection refused: no further information

testSet(com.wzf.RedisTest)  Time elapsed: 2.218 s  <<< ERROR!
org.springframework.data.redis.RedisConnectionFailureException: Unable to connect to Redis; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6379
	at com.wzf.RedisTest.testSet(RedisTest.java:25)
Caused by: io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6379
	at com.wzf.RedisTest.testSet(RedisTest.java:25)
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
Caused by: java.net.ConnectException: Connection refused: no further information

